tests/foreman/api/test_activationkey.py

```diff
@@ -146,6 +164,27 @@ def test_positive_update_1(self):
                 actual = {attr: getattr(act_key, attr) for attr in want.keys()}
                 self.assertEqual(want, actual)
 
+    def test_positive_update_2(self):
+        """@Test: Create activation key providing the initial name, then update
+        its name to another valid name.
+
+        @Assert: Activation key is created, and its name can be updated.
+
+        @Feature: ActivationKey
+
+        """
+        act_key = entities.ActivationKey().create()
+
+        for new_name in valid_names_list():
```

Comment:
It seems `valid_names_list` is pretty redundant method or at least applicable only for test_user module (as there are specific requirements for user login value). Anyway, we are using `valid_data_list()` method for 99% of cases in our code